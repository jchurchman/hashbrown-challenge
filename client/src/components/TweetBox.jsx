import styled from 'styled-components'
import Card from '@material-ui/core/Card';
import CardContent from '@material-ui/core/CardContent';
import Typography from '@material-ui/core/Typography';

const StyledCard = styled(Card)`
  max-width: 40em;
  margin: 0.5em;
`

const TweetBox = ({ text }) => {
  return (
    <StyledCard>
      <CardContent>
        <Typography
          align="center"
          display="block"
          variant="body1"
        >
          {text}
        </Typography>
      </CardContent>
    </StyledCard>
  )
}

export default TweetBox