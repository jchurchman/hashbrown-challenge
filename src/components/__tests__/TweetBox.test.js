import renderer from 'react-test-renderer';
import TweetBox from '../TweetBox';

describe('TweetBox ', () => {
  it('renders as expected ', () => {
    const tree = renderer
      .create(<TweetBox text={'foo'} />)
      .toJSON()

    expect(tree).toMatchSnapshot()
  })
})