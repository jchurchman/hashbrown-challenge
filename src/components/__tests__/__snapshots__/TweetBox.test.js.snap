// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TweetBox  renders as expected  1`] = `
<div
  className="MuiPaper-root MuiCard-root sc-bdnxRM iEuSkT MuiPaper-elevation1 MuiPaper-rounded"
>
  <div
    className="MuiCardContent-root"
  >
    <p
      className="MuiTypography-root MuiTypography-body1 MuiTypography-alignCenter MuiTypography-displayBlock"
    >
      foo
    </p>
  </div>
</div>
`;
